android-libkiwixbuild/app/src/main/cpp/include/zim/tools.h

```diff
@@ -17,8 +17,8 @@
  *
  */
 
-#ifndef ZIM_TOOLS_H
-#define ZIM_TOOLS_H
+
+
```

Comment:
Why remove these lines ?

(And you must remove the whole include files, not few lines in them)

Reply:
I have removed static includes header files now.